src/renderer/screens/ColorPalette/ColorButton.js

```diff
@@ -0,0 +1,85 @@
+/**
```

Comment:
Can you add copyright headers for the new files? You can put yourself as copyright holder, don't need to put the same info there as in the rest of the files (but you can, if you're ok with having your copyright assigned to Keyboardio).